lib/alephant/broker/response/batch.rb

```diff
@@ -35,7 +35,9 @@ def json
               "status"       => component.status,
               "content_type" => component.content_type,
               "body"         => component.content
-            }
+            }.tap do |headers|
+              headers["seq_id"] = component.headers["X-Sequence"] if component.headers["X-Sequence"]
```

Comment:
I have changed this to your suggestion. I thought it was good to keep it smaller, but don't mind.